incubator/cassandra/templates/cassandra-statefulset.yaml

```diff
@@ -103,13 +103,30 @@ spec:
         livenessProbe:
           exec:
             command: [ "/bin/sh", "-c", "nodetool status" ]
+          {{- if not .Values.livenessProbe }}
```

Comment:
is this really needed when you have the default section lower down?

Reply:
True that, thanks for the suggestion.